src/index.js

```diff
@@ -1,2 +1,40 @@
-// Import styles
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225524684-16d21b01-b60d-49fd-a4a1-99e54f1c2e03.png)

- [ ] It would be nice if you use a descriptive name for your branch such as **feature/leaderboard-hit-api** which is much more descriptive than `project-setup` 👍 and make your project professional 😊

Reply:
I did not understand how to rename a branch without closing PR, but I will delete this branch after merging PR and will give new ones more descriptive names.